packages/teleterm/src/ui/DocumentGateway/DocumentGateway.tsx

```diff
@@ -87,8 +110,29 @@ export function DocumentGateway(props: State) {
         </ButtonSecondary>
       </Flex>
       <Text bold>Connect with CLI</Text>
```

Comment:
Those uppercased labels look a bit weird next to a section header that's almost the same size as them.

```suggestion
      <Text typography="h4">Connect with CLI</Text>
```

Reply:
Good idea, looks better